.gitignore

```diff
@@ -1,6 +1,6 @@
 # Houston files
 *.log
-config.js
+/config.js
```

Comment:
There is another config file for tests. I only want the root config to be ignored.